armory/utils/metrics.py

```diff
@@ -326,5 +370,20 @@ def results(self):
                         raise ZeroDivisionError(
                             f"No values to calculate mean in {prefix}_{metric.name}"
                         )
-
+        if computational_resource_dict:
```

Comment:
This `if` is unnecessary; the dict always exists and `for` looping over an empty dict is valid

Reply:
Yup, removed.